### PR TITLE
[#76] [UI] Implement Answer UI: Rating Bar  [Part 1]

### LINKED ISSUE #76: [UI] As a user, I can see different Answer choices
## Why

We need to display Answer choice depending on the type from the API.
 
## Who Benefits?
Prepare all the UI for all different choices:
image

### PR DESCRIPTION
**Pull request by sleepylee**
Part of #76 

## What happened 👀
Implement the Answer widget. Part 1 is to create the basic Rating Bar type.
 
## Insight 📝

- Simply using rating bar for type `like` `star` `heart`
 
## Proof Of Work 📹

https://user-images.githubusercontent.com/13435717/124867249-eb1e4100-dfe7-11eb-8c58-70856e5dcb78.mp4

### CODE REVIEW

File: lib/pages/survey/survey_answer.dart [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+Widget _starRatingBar(int counter) {
+  return RatingBar.builder(
+    initialRating: 3,
+    minRating: 1,
+    unratedColor: Colors.amber.withOpacity(0.4),
+    direction: Axis.horizontal,
+    itemCount: counter,
+    itemPadding: EdgeInsets.symmetric(horizontal: 5.0),
+    itemBuilder: (context, _) => Text("⭐️"),
```
Comment (manh-t) [inline lib/pages/survey/survey_answer.dart:56]: oh, icon text 🤔

**Review by manh-t — APPROVED** [review]

File: lib/pages/survey/survey_answer.dart (hunk 1 above)
Comment (sleepylee) [reply lib/pages/survey/survey_answer.dart:56]: from the design 😛  Mike used Text and add Emoji in 😀

**Comment by sleepylee** [comment]
I'll need to refactor this a bit

**Comment by sleepylee** [comment]
updated with these Rating type:

**Comment by manh-t** [comment]
LGTM

**Review by markgravity — APPROVED** [review]
LGTM 🤖

**Pull request merged**